### PR TITLE
chore: repo-linter-checks

### PR DESCRIPTION
**Pull request by jpvajda**
- This PR resolve the issues reported by repo-linter for OSS policies
- I also made markdown linter fixes
- this also adds a release.yml file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
-[![Community Project header](https://opensource.newrelic.com/static/Community_Project-7c4805883d6396086f907f1c716477cd.png)](https://opensource.newrelic.com/oss-category/#community-project)
+[![Community header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Project.png)](https://opensource.newrelic.com/oss-category/#community-project)
```
[the remark this answers is not in the document]
Comment (zstix) [reply README.md:1]: This is an issue with how repolinter is handling these image paths.

[the remark this answers is not in the document]
Comment (jpvajda) [reply README.md]: Thanks! I'll look into how to fix this in repo-linter 

https://github.com/newrelic/.github/blob/main/docs/repolinter/policy-overviews.md

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+          github-token: ${{ secrets.OPENSOURCE_BOT_TOKEN }}
+          previews: luke-cage-preview
+          script: |
+            const result = await github.repos.updateBranchProtection({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              branch: 'main',
+              required_status_checks: null,
+              restrictions: null,
+              enforce_admins: true,
```
Comment (aswanson-nr) [inline .github/workflows/release.yml:183]: Unless this is required for the third party notices, we don't need to generate a changelog

**Review by aswanson-nr — COMMENTED** [review]
Looks good! I don't think we need the `generate-changelog` job unless that's also required

File: .github/workflows/release.yml (hunk 2 above)
Comment (jpvajda) [reply .github/workflows/release.yml:183]: I actually think a change log is just a good best practice for an OSS repository. So is there any reason not to include this functionality?

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+              required_pull_request_reviews: {
+                dismiss_stale_reviews: true,
+                required_approving_review_count: 1
+              }
+            })
+            console.log("Result:", result)
```
Comment (aswanson-nr) [inline .github/workflows/release.yml:120]:
```suggestion
      - name: Re-enable "required_pull_request_reviews" branch protection
        id: enable-branch-protection
        if: always()
        uses: actions/github-script@v1
        with:
          github-token: ${{ secrets.OPENSOURCE_BOT_TOKEN }}
          previews: luke-cage-preview
          script: |
            const result = await github.repos.updateBranchProtection({
              owner: context.repo.owner,
              repo: context.repo.repo,
              branch: 'main',
              required_status_checks: null,
              restrictions: null,
              enforce_admins: true,
              required_pull_request_reviews: {
                dismiss_stale_reviews: true,
                required_approving_review_count: 1
              }
            })
            console.log("Result:", result)
```

Comment (aswanson-nr) [inline .github/workflows/release.yml:120]:
```suggestion
            const result = await github.repos.updateBranchProtection({
              owner: context.repo.owner,
              repo: context.repo.repo,
              branch: 'main',
              required_status_checks: {
                strict: false,
                contexts: [
                  'Gatsby Build Service - instant-observability-website'
                ]
              },
              restrictions: null,
              enforce_admins: true,
              required_pull_request_reviews: {
                dismiss_stale_reviews: true,
                required_approving_review_count: 1
              }
            })
            console.log("Result:", result)
```

File: .github/workflows/release.yml (hunk 2 above)
Comment (aswanson-nr) [reply .github/workflows/release.yml:183]: I can agree with that! Just two more comments then to get the branch protection right

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
```
Comment (aswanson-nr) [inline .github/workflows/release.yml:189]:
```suggestion
            const result = await github.repos.updateBranchProtection({
              owner: context.repo.owner,
              repo: context.repo.repo,
              branch: 'main',
              required_status_checks: {
                strict: false,
                contexts: [
                  'Gatsby Build Service - instant-observability-website'
                ]
              },
              restrictions: null,
              enforce_admins: true,
              required_pull_request_reviews: {
                dismiss_stale_reviews: true,
                required_approving_review_count: 1
              }
            })
            console.log("Result:", result)
```

File: .github/workflows/release.yml (hunk 3 above)
Comment (jpvajda) [reply .github/workflows/release.yml:120]: @aswanson-nr is this any different? I'm seeing this error when trying to commit 

` Suggestion cannot be identical to original text.`

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Agreed on the changelog! Just need a few changes to the branch protection

File: .github/workflows/release.yml (hunk 3 above)
Comment (aswanson-nr) [reply .github/workflows/release.yml:120]: Let me take a look

**Comment by aswanson-nr** [comment]
Not sure what happened with those comments, but this is what needs to be added to the "re-enable branch protection"
```
required_status_checks: {
  strict: false,
  contexts: [
     'Gatsby Build Service - instant-observability-website'
  ]
},
```

**Comment by jpvajda** [comment]
@aswanson-nr I made the suggested change, let me know what you think. Thanks for the input here!

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+        uses: actions/github-script@v1
+        with:
+          github-token: ${{ secrets.OPENSOURCE_BOT_TOKEN }}
+          previews: luke-cage-preview
+          script: |
+            const result = await github.repos.updateBranchProtection({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              branch: 'main',
+              required_status_checks: null,
```
Comment (aswanson-nr) [inline .github/workflows/release.yml:186]: You may need to do this locally, sometimes the formatting get's messed up with these suggestions
```suggestion
              required_status_checks: {
                strict: false,
                contexts: [
                    'Gatsby Build Service - instant-observability-website'
                ]
              },
```

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**